src/styles/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  .stars--4 &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
The modifier class `.stars--4` should be defined at the same level as `.stars` to maintain the flat structure of SCSS and BEM. Nested modifiers can make the code harder to read and maintain. Consider moving this modifier to the root level of the SCSS file.